tests/format/js/import-reflection/__snapshots__/format.test.js.snap

```diff
@@ -243,11 +243,11 @@ exports[`valid-from-as-default-module-binding-escaped.mjs [flow] format 1`] = `
 `;
 
 exports[`valid-from-as-default-module-binding-escaped.mjs [meriyah] format 1`] = `
-"Import source must be string (1:17)
+"Unexpected token: 'reserved if strict' (1:17)
 > 1 | import module \\u0066rom from "./module.wasm";
     |                 ^
   2 |
-Cause: [1:17]: Import source must be string"
+Cause: [1:17]: Unexpected token: 'reserved if strict'"
```

Comment:
@3cp What this error mean....

Reply:
After first two tokens "import module", the parse now check if following token is indeed FromKeyword. It's complaining `\\u0066rom` is not FromKeyword.

Previously it silently ignored missing FromKeyword, then parser expects a string literal for the module source, hence "Import source must be string" error.

Reply:
Oh, I guess your question is 'reserved if strict', let me check why the escaped unicode mapped to that type.

Reply:
It should map to 'escaped keyword'. Something is broken.

Reply:
Ok, got it, "from" FromKeyword is not reserved, nor future reserved. So `\u0066rom` is not parsed as EscapedReserved or EscapedFutureReserved, it's parsed as AnyIdentifier.

AnyIdentifier is mapped to 'reserved if strict', probably for some other cases.

Reply:
I did a test, after I turn FromKeyword into Reserved, `FromKeyword        = 116 | Contextual | Reserved,`, it began to show error `SyntaxError: [1:16]: Unexpected token: 'escaped keyword'`.

Reply:
Not important, just curious, thanks for the explanation.